[DOCS] Add instructions to turn off Endpoint diagnostic data (#1641)
elastic/security-docs: docs/getting-started/endpoint-diagnostic-data.asciidoc

[[endpoint-diagnostic-data]]
= Turn off {endpoint-sec} diagnostic data

By default, {endpoint-sec} streams diagnostic data to your cluster, which Elastic uses to tune protection features. You can stop producing this diagnostic data by configuring the advanced settings in the {endpoint-sec} integration policy.

> NOTE: Kibana also collects usage telemetry, which includes {endpoint-sec} diagnostic data. You can control Kibana telemetry in {kibana-ref}/telemetry-settings-kbn.html[Advanced Settings].

Reviewer: Are we officially calling the telemetry feature "Kibana telemetry"? I thought it was just "telemetry".

Author: @nastasha-solomon This wasn't intended as an official name of the feature, just a short description. Maybe "telemetry in Kibana"?
```suggestion
NOTE: Kibana also collects usage telemetry, which includes {endpoint-sec} diagnostic data. You can control telemetry in Kibana in {kibana-ref}/telemetry-settings-kbn.html[Advanced Settings].
```

Reviewer 2: I _think_ it's just `telemetry` too.

Author: Revised the suggestion for just "telemetry"

. In the {security-app}, go to *Manage* -> *Endpoints* to view the Endpoints list.

> . On endpoints where you want to disable diagnostic data, click the integration policy in the *Policy* column.

Reviewer: ```suggestion
. Go to the endpoint that you want to disable diagnostic data on, then click the integration policy in the *Policy* column.
```

Reviewer: Mainly leaving this to indicate that endpoint should be singular.

Author: @nastasha-solomon Gotcha, I think we can stay close to the original and still revise to singular:
```suggestion
. On the endpoint where you want to disable diagnostic data, click the integration policy in the *Policy* column.
```

Reviewer 2: Revision to @nastasha-solomon suggestion: 

> Go to the endpoint for which you want to disable diagnostic data, then click the integration policy in the **Policy** column.

Can also use "locate the endpoint" in lieu of "go to".
